features/godogs_test.go

```diff
@@ -784,3 +788,100 @@ func heeftDePersoonGeenRechtOpKinderopvangtoeslag(ctx context.Context) error {
 
 	return nil
 }
+
+func isGelijkAan(ctx context.Context, variable, value string) error {
+	result, ok := ctx.Value(resultCtxKey{}).(model.RuleResult)
+	assert.True(godog.T(ctx), ok)
+
+	v, ok := result.Output[variable]
+	if !ok {
+		return fmt.Errorf("expected '%s' to be in the output, but it wasn't found", variable)
+	}
+
+	// Parse the expected value (handle bool, numeric, string)
+	var expectedValue any = value
+	if value == "true" || value == "false" {
```

Comment:
hier zou je: `strconv.ParseBool(v)` kunnen gebruiken. 
Dan werkt een `FALSE` of `False` ook